src/Api/Dto/Cover.php

```diff
@@ -49,10 +49,12 @@
  *                      {
  *                          "name" = "identifiers",
  *                          "in" = "query",
- *                          "description" = "A list of identifiers of {type}",
+ *                          "description" = "A list of identifiers of {type}. Maximum number os identifiers per reqeust is %d",
  *                          "required" = true,
  *                          "schema" : {
  *                              "type": "array",
+ *                              "maxLength": 200,
```

Comment:
Should 200 be substituted with `%d`?

Reply:
Tried doing that. But then the annotation reader died. Have changed it to `"maxLength": "DECORATED_ENV_VALUE",`